lib/socket.js

```diff
@@ -65,7 +65,7 @@ class Transport extends EventEmitter {
     this._buffer += chunk;
 
     try {
-      this._buffer = jsrs.parseNetworkPackets(this._buffer, packets);
+      this._buffer = json5.parseNetworkPackets(this._buffer, packets);
```

Comment:
That's a valid point and I though of it too. Despite that, we literally have the network messages parser being a part of the JSON5 parser, tightly coupled with its internals. Do you have any suggestions about how we should handle this?